src/lib/StorageFileApi.ts

```diff
@@ -62,10 +62,16 @@ export class StorageFileApi {
       | URLSearchParams
       | string,
     fileOptions?: FileOptions
-  ): Promise<{
-    data: { Key: string; path: string; bucketId?: string } | null
-    error: Error | null
-  }> {
+  ): Promise<
+    | {
+        data: { Key: string; path: string; bucketId?: string }
+        error: null
+      }
+    | {
+        data: null
+        error: StorageError
+      }
+  > {
```

Comment:
@alaister is this what you meant by " updating the return types to match how we're doing returns now"?

Reply:
Yes perfect 👌